### PR TITLE
readme: mention -a action + add example

### PR DESCRIPTION
**Pull request by Quasilyte**
Example includes type-based filtering of matched results.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +50,8 @@ Regexes can also be used to match certain identifier names only. The
 
 The nodes resulting from applying the commands will be printed line by
 line to standard output.
+
+Example of using `-a` argument is filtering matched expressions by type.
+
+       gogrep -x '$x + $y'                   // will match both numerical and string "+" operations
+       gogrep -x '$x + $y' -a 'type(string)' // matches only concatenations
```
Comment (mvdan) [inline README.md]: string concatenations

File: README.md [hunk 2]
```diff
@@ -49,3 +50,8 @@ Regexes can also be used to match certain identifier names only. The
 
 The nodes resulting from applying the commands will be printed line by
 line to standard output.
+
+Example of using `-a` argument is filtering matched expressions by type.
```
Comment (mvdan) [inline README.md]: Here are two simple examples of the `-a` operand:

File: README.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ A command is of the form "-A pattern", where -A is one of:
        -x  find all nodes matching a pattern
        -g  discard nodes not matching a pattern
        -v  discard nodes matching a pattern
+       -a  attribute discard nodes without an attribute
```
Comment (mvdan) [inline README.md]: > -a    filter nodes by certain attributes

File: README.md [hunk 4]
```diff
@@ -17,6 +17,7 @@ A command is of the form "-A pattern", where -A is one of:
        -x  find all nodes matching a pattern
        -g  discard nodes not matching a pattern
        -v  discard nodes matching a pattern
+       -a filter nodes by certain attributes
```
Comment (mvdan) [inline README.md]: nit: please add a space to keep the alignment

File: README.md [hunk 5]
```diff
@@ -49,3 +50,8 @@ Regexes can also be used to match certain identifier names only. The
 
 The nodes resulting from applying the commands will be printed line by
 line to standard output.
+
+Here are two simple examples of the -a operand:
+
+       gogrep -x '$x + $y'                   // will match both numerical and string "+" operations
```
Comment (mvdan) [inline README.md]: `s/will match/matches/`

This way the two sentences are more consistent.

File: README.md [hunk 6]
```diff
@@ -49,3 +50,8 @@ Regexes can also be used to match certain identifier names only. The
 
 The nodes resulting from applying the commands will be printed line by
 line to standard output.
+
+Here are two simple examples of the -a operand:
```
Comment (mvdan) [inline README.md]: `s/are two/is a/`, since there's only one `-a` command.

**Comment by mvdan** [comment]
Bump @Quasilyte? Otherwise I'll just amend and merge myself.

**Comment by Quasilyte** [comment]
> Otherwise I'll just amend and merge myself.

@mvdan, I've faced inconveniences when editing PR sent by github web UI, so it would be great if you do that.

**Comment by mvdan** [comment]
Merged.

**Pull request closed without merging**